WebDev.md

```diff
@@ -53,3 +53,5 @@ Remember, the deadline for this week's task is June 7th, 2022, at 23:59.
 <!-- - Your Name - [Repo Name](Link) [Site](Site Link) -->
 
 <!-- - Sanyu Daver - [Tic Tac](https://github.com/sanyud/TicTac) [Site](www.copsiitbhu.co.in) -->
+
+<!-- Atul Yadav - [Hangman](https://github.com/atulyadav745/Hangman) [Site](https://atulyadav745.github.io/Hangman/) -->
```

Comment:
The changes here are commented out just like the example given above :) You need to remove those brackets to uncomment it.